hal-x86_64/src/interrupt/mod.rs

```diff
@@ -55,7 +55,7 @@ pub fn init<H: Handlers<Registers>>() -> Control {
     tracing::info!("configuring 8259 PIC interrupts...");
 
     unsafe {
-        PIC.set_irq_addresses(0x20, 0x28);
+        PIC.set_irq_addres(0x20, 0x28);
```

Comment:
why `addres`? It seems like you're trying to normalize to `addr` in most other places.

Reply:
this is a find-and-replace accident (replacing "address" with "addr" turns addresses" into "addres"). thanks for noticing that!